src/utils/ProfileUtils.ts

```diff
@@ -134,6 +122,7 @@ export async function installModsToProfile(
     }
 
     throwForR2Error(await ProfileModList.saveModList(profile, profileMods));
+    progressCallback('', 100);
```

Comment:
Would be a lot safer to use `"Copying mods to profile: 100%"` instead of an empty string.

Reply:
Sure, even if it's not used at this point, it is better for the future developer who might assume it having the correct status string value.